src/System.Net.Requests/src/System.Net.Requests.csproj

```diff
@@ -49,6 +49,10 @@
     <Compile Include="$(CommonPath)\System\NotImplemented.cs">
       <Link>Common\System\NotImplemented.cs</Link>
     </Compile>
+    <ProjectReference Include="..\..\System.Net.Http\pkg\System.Net.Http.pkgproj">
```

Comment:
I believe this is correct but I cannot recall for sure. I remember at some points we needed to reference the csproj and in some cases we reference the pkgproj. If the CI passes with the pkgproj we can keep it like it is.